device_doctor/lib/src/mac.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:process/process.dart';
+
+import 'health.dart';
+import 'utils.dart';
+
+/// The health check for Mac swarming user auto login.
```

Comment:
Can we expand this comment to tell that this is a requirement to run GUI application on mac?

Reply:
Updated.